include/swift/Parse/Lexer.h

```diff
@@ -24,6 +24,8 @@
 #include "llvm/ADT/SmallVector.h"
 
 namespace swift {
+  uint32_t validateUTF8CharacterAndAdvance(const char *&Ptr, const char *End);
+  bool EncodeToUTF8(unsigned CharValue, SmallVectorImpl<char> &Result);
```

Comment:
If you're going to publish these—and I'm not convinced that's a good idea yet, see below—please give them proper doc comments, and change the second to `encodeToUTF8` to follow dominant naming conventions.